metadata.json

```diff
@@ -25,6 +25,7 @@
     { "operatingsystem": "Scientific", "operatingsystemrelease": [ "6", "7" ] },
     { "operatingsystem": "Fedora",     "operatingsystemrelease": [ "19" ] },
     { "operatingsystem": "Debian",     "operatingsystemrelease": [ "6", "7" ] },
-    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] }
+    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] },
+    { "operatingsystem": "Windows",    "operatingsystemrelease": [ "7", "2012 R2" ] }
```

Comment:
Puppet's list of what they support is at https://docs.puppetlabs.com/guides/install_puppet/install_windows.html#supported-versions
My setting should work with Vista - 8 and Server 2008 - 2012 R2. Server 2003 would require different defaults because of the difference in the data directory (https://docs.puppetlabs.com/pe/latest/install_windows.html#data-directory)

What are your thoughts on which ones should be in this file?